src/fonduer/meta.py

```diff
@@ -16,49 +15,35 @@ def new_sessionmaker():
     # http://oddbird.net/2014/06/14/sqlalchemy-postgres-autocommit/
     # Otherwise any e.g. query starts a transaction, locking tables... very
     # bad for e.g. multiple notebooks open, multiple processes, etc.
-    if Meta.postgres and Meta.ready:
+    try:
         engine = create_engine(
             Meta.conn_string,
             client_encoding="utf8",
             use_batch_mode=True,
             isolation_level="AUTOCOMMIT",
         )
-    else:
+        engine.connect()
+    except Exception:
         raise ValueError(
             "Meta variables have not been initialized with "
             "a valid postgres connection string."
+            "Use the form: "
+            "postgresql://<user>:<pw>@<host>:<port>/<database_name>"
```

Comment:
So this means we need to change `postgres` to `postgresql` in all the tutorials as well, right?

Reply:
yes